### DISCUSSION TITLE
Some tips for a more beautiful Web

### DISCUSSION THREAD

**Discussion by Odi-3** [elementary/browser#108] [Ideas]
### Problem

I think some style changes would make the Web more modern.
As always, I would like to specify that it is only my idea and that suggestions and comparisons are welcome.

### Proposal

- Shorter and thicker address bar;
- Toggle the "+" button to add new tabs on the window bar;
- Remove the tab bar when only one tab is open;
- Make tabs fit window width (this may apply to all applications).

**Comment by Odi-3** [comment]
P.S. AppCenter address bar is a perfect example

**Comment by janxkoci** [comment]
Just FYI, you are describing what GNOME already does. So in principle, you could just use Adwaita (I guess), which already does all of these things.

But elementary design philosophy explicitly differs on these points so I don't think they will consider it. You can read the HIG (human interface guidelines) of both projects to learn more.

**Reply by Odi-3** [reply]
Yes, in fact the GNOME style respects these points perfectly. This does not mean that the same thing cannot be done but in a different way.

**Comment by danirabbit** [comment]
I think there's an opportunity to re-evaluate some of the upstream conditionals.

Just FYI to the OP, an issue report should contain a single, actionable item. For broader ideas please use discussions
